models/bamf_nnunet_ct_kidney/meta.json

```diff
@@ -0,0 +1,135 @@
+        },
+        "analyses": {
+            "title": "Quantitative Analyses",
+            "text": "The model's performance was assessed using the Dice Coefficient and Normalized Surface Distance (NSD) with tolerance 7mm, as specified in the CT Kidney segmentation task in the Medical Segmentation Decathlon challenge. The model was used to segment cases from the IDC collection TCGA-KIRC [1]. Seven of those cases were reviewed and corrected by a board-certified radiologist and a non-expert. The analysis is published here [2]",
+            "tables": [
+                {
+                    "label": "Label-wise metrics (mean (standard deviation)) between AI derived and manually corrected CT kidney annotations",
+                    "entries": {
+                        "Dice: Radiologist": "0.93 (0.22)",
+                        "NSD: Radiologist": "0.91 (0.23)",
+                        "Dice: Non-expert": "0.99 (0.06)",
+                        "NSD: Non-expert": "0.98 (0.09)"
+                    }
+                }
+            ],
+            "references": [
+                {
+                    "label": "TCGA-KIRC",
+                    "uri": "https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=5800386"
+                },
+                {
+                    "label": "The AIMI Initiative: AI-Generated Annotations for Imaging Data Commons Collections",
+                    "uri": "https://arxiv.org/abs/2310.14897"
+                }
+            ]
+        },
+        "evaluation": {
+            "title": "Evaluation Data",
+            "text": "The model was used to segment cases from the IDC [1] collection TCGA-KIRC [1]. Approximately 20% of those cases were randomly selected to be reviewed and corrected by a board-certified radiologist. The model predictions, and radiologist corrections are published on zenodo [3]",
+            "references": [
+                {
+                    "label": "Imaging Data Collections (IDC)",
+                    "uri": "https://datacommons.cancer.gov/repository/imaging-data-commons"
+                },
+                {
+                    "label": "TCGA-KIRC",
+                    "uri": "https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=5800386"
+                },
+                {
+                    "label": "Image segmentations produced by the AIMI Annotations initiative",
+                    "uri": "https://zenodo.org/records/10009368"
+                }
+            ]
+        },
+        "training": {
+            "title": "Training Data",
+            "text": "The training dataset consists of  489 CT kidney annotations taken from KiTS 2023 dataset [1] (N=489) and TCGA-KIRC Dataset [2] (N=45).",
```

Comment:
If the training dataset contains demographic information that helps the reader to assess the suitability of the model for his use case then I suggest to mention this in the text. After all, I wouldn't expect a reader to click on any link so with the additional information we can ideally motivate that initiative 😀